src/components/views/MoviesSummary.vue

```diff
@@ -1,15 +1,21 @@
 <script>
 import { defineComponent } from 'vue';
-import MovieCard from '@/components/views/MovieCard.vue'
+import MovieCard from './MovieCard.vue';
+import { mapState } from "pinia";
+import movies from '@/stores/moviesStore'
 
 export default defineComponent({
   components: { MovieCard },
-  props: {
-    movies: {
-      type: Array,
-      required: true,
+  data() {
```

Comment:
Empty? Can be removed 😄

Reply:
Yess... Actually I should prepare for myself a checklist of things to be deleted before each PR submission -- commiting redundant code is my bane 🙄 😉